Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env groovy
```

Comment:
Creating a PR against jenkins-seed is our flow for getting pipelines into Jenkins at the moment. Those links are for context and learning about our CI setup.

I skipped this flow and committed the api-java pipeline line item to jenkins-seed master and added a stub Jenkinsfile to this PR, as Alan said, this time.